docs/openapi/schemas.yaml

```diff
@@ -1517,3 +1525,62 @@ ImportAssistantResponse:
     choices:
       - message:
           content: '.breadcrumbs'
+ApiKeyRequest:
+  type: object
+  required:
+    - name
+    - feature
```

Comment:
Should `feature` be `features` instead, and an array? Thinking ahead to future use cases.

Reply:
Do we want to have the same api-key for imports and organizations? Maybe yes. I could change it to an array.